charts/hlf-k8s/values.yaml

```diff
@@ -87,27 +120,27 @@ peer:
     ## These secrets should contain the Orderer crypto materials and credentials
     peer:
       ## Certificate, saved under key 'cert.pem'
-      cert: hlf-idcert
+      cert: hlf-msp-cert-user
       ## Key, saved under 'key.pem'
-      key: hlf-idkey
+      key: hlf-msp-key-user
```

Comment:
Yes, this is the cert and they key. We need to create these secrets because they're consumed by hlf-peer and hlf-ord, e.g. https://github.com/helm/charts/blob/master/stable/hlf-peer/values.yaml#L86-L89

Reply:
Ok, then it is not possible for us to merge them in a single file.